.gitignore

```diff
@@ -45,3 +45,22 @@ sdk/target
 */.idea/
 */*/.idea/
 build/**
+
+execution-engine/Cargo.lock
```

Comment:
Any reason not to use `*/Cargo.lock` here?

Reply:
I wanted to be sure that I would not ignore more files than I needed to but your suggestion is more future-proof.